### PR TITLE
Modify getting started example for version 4.0

### PR DESCRIPTION
**Pull request by beenanner**

### CODE REVIEW

**Comment by ilanbiala** [comment]
@beenanner try closing and reopening the PR to trigger a new build.

**Comment by beenanner** [comment]
Thanks @ilanbiala I closed and reopened the pull-request

File: docs/getting-started.md [hunk 1]
```diff
@@ -37,7 +40,7 @@ To run individual tasks, use `gulp <task> <othertask>`.
 
 You have an empty gulpfile and everything is installed. How do you REALLY get started? Check out the [recipes](recipes) and the [list of articles](README.md#articles) for more information.
 
-## .src, .watch, .dest, CLI args - How do I use these things?
+## .src, .watch, .dest, .parallel, .serial, CLI args - How do I use these things?
```
Comment (phated) [inline docs/getting-started.md]: `.series` is the method name, not serial

Comment (beenanner) [inline docs/getting-started.md]: Good catch @phated I've fixed the typo

**Comment by phated** [comment]
Looks good. Thanks!

**Pull request merged**